src/js/components/wizard.js

```diff
@@ -0,0 +1,110 @@
+import FluxComponent from 'flummox/component';
+import React from 'react';
+
+
+let Wizard = React.createClass({
+  render() {
+    let steps = this.props.children.map(function(step, index) {
+      return React.cloneElement(step, {
+        key: index,
+        isActive: index === this.props.activeStep
+      });
+    }, this);
+
+    return (
+      <section className="wizard">
+        <Wizard.ProgressBar wizard={this}/>
+        {steps}
+        <Wizard.Menu wizard={this}/>
+      </section>
+    );
+  }
+});
+
+
+Wizard.ProgressBar = React.createClass({
+  propTypes: {
+    wizard: React.PropTypes.instanceOf(Wizard).isRequired
+  },
+
+  handleClick(num) {
+    return () => { this.props.wizard.props.flux.getActions('wizard').goToStep(num) }  
+  },
+
+  render() {
+    return (
+      <menu className="wizard--progress-bar">
+        {this.props.wizard.props.children.map(
+          (step, index) => <Wizard.ProgressItem handleClick={this.handleClick(index)} key={index} step={step}/>
+        )}
+      </menu>
+    );
+  }
+});
+
+
+Wizard.ProgressItem = React.createClass({
+  render() {
+    return <button onClick={this.props.handleClick}>{this.props.step.props.title}</button>;
+  }
+});;
+
+
+Wizard.Step = React.createClass({
+  propTypes: {
+    isActive: React.PropTypes.bool,
+    title: React.PropTypes.string.isRequired
+  },
+  stepStyle() {
+    return { display: this.props.isActive ? 'block': 'none' }
+  },
+  render() {
+    return (
```

Comment:
I prefer visually matching the opening and closing tags' indentation, but that's fine I guess.